pytorch/training/docker/1.8.0/py3/cu111/Dockerfile.gpu

```diff
@@ -0,0 +1,259 @@
+FROM nvidia/cuda:11.1.1-base-ubuntu18.04
+
+LABEL maintainer="Amazon AI"
+LABEL dlc_major_version="1"
+
+ARG PYTHON_VERSION=3.6.13
+ARG OPEN_MPI_VERSION=4.0.1
+ARG CUBLAS_VERSION=11.3.0.106
+ARG OPEN_MPI_PATH=/home/.openmpi
+ARG CUDA_HOME=/usr/local/cuda
+ARG CONDA_PREFIX=/opt/conda
+ARG METIS=metis-5.1.0
+ARG RMM_VERSION=0.15.0
+
+# The smdebug pipeline relies for following format to perform string replace and trigger DLC pipeline for validating
+# the nightly builds. Therefore, while updating the smdebug version, please ensure that the format is not disturbed.
+ARG SMDEBUG_VERSION=1.0.1
+
+# Python won’t try to write .pyc or .pyo files on the import of source modules
+# Force stdin, stdout and stderr to be totally unbuffered. Good for logging
+ENV PYTHONDONTWRITEBYTECODE=1
+ENV PYTHONUNBUFFERED=1
+ENV LD_LIBRARY_PATH="/usr/local/lib:${LD_LIBRARY_PATH}"
+ENV LD_LIBRARY_PATH="/opt/conda/lib:${LD_LIBRARY_PATH}"
+ENV PYTHONIOENCODING=UTF-8
+ENV LANG=C.UTF-8
+ENV LC_ALL=C.UTF-8
+ENV PATH /opt/conda/bin:$PATH
+ENV TORCH_CUDA_ARCH_LIST="3.7 5.0 7.0+PTX 8.0"
+ENV TORCH_NVCC_FLAGS="-Xfatbin -compress-all"
+ENV CUDNN_VERSION=8.0.5.39
+ENV NCCL_VERSION=2.8.4
+ENV HOROVOD_VERSION=0.21.3
+ENV DGLBACKEND=pytorch
+ENV CMAKE_PREFIX_PATH="$(dirname $(which conda))/../"
+ENV SAGEMAKER_TRAINING_MODULE=sagemaker_pytorch_container.training:main
+ENV MANUAL_BUILD=0
+
+ARG PT_TRAINING_URL=https://aws-pytorch-binaries.s3-us-west-2.amazonaws.com/r1.8.0_aws/20210306-000727/b5cc095ca218fb9eab43948dc83c7a020ab1db26/gpu/torch-1.8.0-cp36-cp36m-manylinux1_x86_64.whl
+ARG PT_TORCHVISION_URL=https://torchvision-build.s3-us-west-2.amazonaws.com/1.8.0/gpu/torchvision-0.9.0-cp36-cp36m-linux_x86_64.whl
+ARG SMD_MODEL_PARALLEL_URL=https://sagemaker-distributed-model-parallel.s3.amazonaws.com/pytorch-1.7/build-artifacts/2021-01-19-23-41/smdistributed_modelparallel-1.2.0-cp36-cp36m-linux_x86_64.whl
```

Comment:
Binary update for SM model parallel library for PT1.8?

Reply:
@rahul003 Do we have a updated model parallel library here?